ONSdigital/response-operations-ui
# Sample to ce page

# What and why?

This PR moves the sample functionality out of the 'sample and ci page' and puts it on the main page for a collection exercise.
A count of the collection instruments linked for an EQ exercise (or uploaded for a SEFT exercise) was added to the instruments part of the collection exercise screen.

Finally it tidies up some of the tests, combining multiple tests into a single case.  This is useful for us as each view test (that craetes a `test_client` is expensive to start, but cheap to tweak once you're running the test)

# How to test?

- Run both EQ and SEFT and BRES exercises and make sure they work
- Upload a large sample (1-3k in dev should do it) and make sure the sample load progress still works

# Trello
